exercises/exercise_02/bad/fortran_ml_mod.f90

```diff
@@ -12,36 +13,36 @@ module ml_mod
 
   ! Set working precision for reals
   integer, parameter :: wp = sp
-   
+
   contains
 
   subroutine ml_routine(in_data, out_data)
 
     ! Set up Fortran data structures
     real(wp), dimension(5), target, intent(in)  :: in_data
     real(wp), dimension(5), target, intent(out) :: out_data
- 
+
     ! Set up Torch data structures
     ! The net, a vector of input tensors, and a vector of output tensors
-    integer, dimension(1) :: tensor_layout = [1]
+    integer, parameter, dimension(1) :: tensor_layout = [1]
```

Comment:
This is no longer needed, right?

Reply:
Yep, this updates the existing files to satisfy Fortitude, but does not make any FTorch API updates - I leave this as a separate issue.